drops/azure_edge_extensions_rag_on_edge.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Retrieval-Augmented Generation on Edge with Azure Arc & Azure IoT Operations",
+  "Summary": "This Drop provides a Retrieval-Augmented Generation(RAG) on edge pattern that facilitates text searches within files utilizing a local vector-based approach Chroma DB and an edge Small/Large Language Model.",
+  "Description": "RAG on Edge is a tool to perform text searches within files using a vector-based approach, and generate a readable response based on the search result with Large/Small Language Model.
+The solution is independent from cloud services, and can be deployed to the edge device. The messaging system follows a pub/sub pattern, ensuring adaptability to diverse edge computing scenarios.",
+  "Cover": "",
+  "Authors": [
+    {
+      "Name": "Cheng Chen",
+      "Link": "https://www.linkedin.com/in/cheng-chen-a8496850/"
+    }
+  ],
+  "Source": "https://github.com/Azure-Samples/azure-edge-extensions-retrieval-augmented-generation",
+  "Type": "sample_app",
+  "Difficulty": "Medium",
+  "ProgrammingLanguage": [
+    "Python",
+    "Shell"
+  ],
+  "Products": [
+    "Azure IoT Operations",  
+    "Arc"

```

Comment:
Kubernetes? Chroma DB?

Reply:
Yes, The pattern is also verified with Phi2/Phi3 and Llama2 models. I will add Phi model, Llama model, ChromaDB to the products section